cli/cliui/agent.go

```diff
@@ -2,31 +2,21 @@ package cliui
 
 import (
 	"context"
-	"fmt"
 	"io"
-	"os"
-	"os/signal"
-	"sync"
 	"time"
 
-	"github.com/briandowns/spinner"
-	"github.com/muesli/reflow/indent"
-	"github.com/muesli/reflow/wordwrap"
+	"github.com/google/uuid"
 	"golang.org/x/xerrors"
 
 	"github.com/coder/coder/codersdk"
 )
 
-var (
-	AgentStartError   = xerrors.New("agent startup exited with non-zero exit status")
-	AgentShuttingDown = xerrors.New("agent is shutting down")
-)
+var AgentShuttingDown = xerrors.New("agent is shutting down")
```

Comment:
nit: does it have to be exposed?
nit2: `ErrAgentShuttingDown` ?

Reply:
Ah, you're right. Seems we're not testing for the error currently so we can unexport it. 👍🏻